be/src/Project/Web/Seller/Seller.Web/Areas/Products/ApiControllers/CategoriesApiController.cs

```diff
@@ -49,21 +50,38 @@ public async Task<IActionResult> Index([FromBody] SaveCategoryRequestModel model
             var token = await HttpContext.GetTokenAsync(ApiExtensionsConstants.TokenName);
             var language = CultureInfo.CurrentUICulture.Name;
 
-            var categoryId = await this.categoriesRepository.SaveAsync(
-                token, language, model.Id, model.ParentCategoryId, model.Name, model.Files.Select(x => x.Id.Value), model.Schema, model.UiSchema);
+            var categoryId = await _categoriesRepository.SaveAsync(
+                token, language, model.Id, model.ParentCategoryId, model.Name, model.Files.Select(x => x.Id.Value), model.Schema, model.UiSchema, model.Order);
 
-            return this.StatusCode((int)HttpStatusCode.OK, new { Id = categoryId, Message = this.productLocalizer.GetString("CategorySavedSuccessfully").Value });
+            return StatusCode((int)HttpStatusCode.OK, new { Id = categoryId, Message = _productLocalizer.GetString("CategorySavedSuccessfully").Value });
+        }
+
+        [HttpPost]        
+        public async Task<IActionResult> Order([FromBody] SaveCategoryOrderRequestModel model)
+        {
+            var token = await HttpContext.GetTokenAsync(ApiExtensionsConstants.TokenName);
+            var language = CultureInfo.CurrentCulture.Name;
+
+            var category = await _categoriesRepository.GetCategoryAsync(token, language, model.Id);
+
+            if(category is not null) 
+            {
+                await _categoriesRepository.SaveAsync(
+                    token, language, category.Id, category.ParentId, category.Name, category.Files.OrEmptyIfNull(), category.Schema, category.UiSchema, model.Order);
+            }
+
+            return StatusCode((int)HttpStatusCode.OK, new { Id = model.Id });
```

Comment:
**suggestion:** I think that passing the Id is not necessary here and only the StatusCode is enough